src/commands/mod.rs

```diff
@@ -46,6 +46,40 @@ fn represents_several_files(files: &[PathBuf]) -> bool {
     files.iter().any(is_non_empty_dir) || files.len() > 1
 }
 
+/// Builds a suggested output file in scenarios where the user tried to compress
+/// a folder into a non-archive compression format, for error message purposes
+///
+/// E.g.: `build_suggestion("file.bz.xz", ".tar")` results in `Some("file.tar.bz.xz")`
+fn build_suggestion(path: &Path, suggested_extension: &str) -> Option<String> {
```

Comment:
Could you change the name of this function to be more specific? Ouch currently does various suggestions inside of this mod.rs, this one is just one of them.

Reply:
Done